### PR TITLE
Improvements to Cwrapper

### PR DESCRIPTION
**Pull request by ShikharJ**
Relevant: #1247

### CODE REVIEW

**Comment by ShikharJ** [comment]
Ping @isuruf.

File: symengine/cwrapper.h [hunk 1]
```diff
@@ -124,6 +124,15 @@ void basic_const_EulerGamma(basic s);
 void basic_const_Catalan(basic s);
 void basic_const_GoldenRatio(basic s);
 
+//! Use these functions to get the use of positive, negative or unsigned
+//! infinity as basic.
+void infty_Inf(basic s);
```
Comment (isuruf) [inline symengine/cwrapper.h]: I think `basic_const_Inf` is better for the name.

File: symengine/cwrapper.h [hunk 2]
```diff
@@ -124,6 +124,15 @@ void basic_const_EulerGamma(basic s);
...
+void infty_NegInf(basic s);
+void infty_ComplexInf(basic s);
+
+//! Use this function to get the use of Nan as basic.
+void nan_Nan(basic s);
```
Comment (isuruf) [inline symengine/cwrapper.h]: `basic_const_nan`

File: symengine/cwrapper.h (hunk 1 above)
Comment (isuruf) [inline symengine/cwrapper.h]: `basic_const_infinity`, `basic_const_neginfinity`, `basic_const_complex_infinity`,

**Comment by ShikharJ** [comment]
Updated @isuruf. Please take a look.

**Comment by isuruf** [comment]
Thanks

**Pull request merged**